WordPress/src/main/java/org/wordpress/android/ui/history/HistoryDetailsContainerFragment.java

```diff
@@ -71,9 +75,25 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
         return rootView;
     }
 
-    @Override
-    public void onViewCreated(final FullScreenDialogController controller) {
-        mDialogController = controller;
+    @Override public void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setHasOptionsMenu(true);
+    }
+
+    @Override public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
+        super.onCreateOptionsMenu(menu, inflater);
+        inflater.inflate(R.menu.revision_details, menu);
+    }
+
+    @Override public boolean onOptionsItemSelected(MenuItem item) {
```

Comment:
```suggestion
    @Override
    public boolean onOptionsItemSelected(MenuItem item) {
```

Can we move the `@Override` annotation to a separate line for code style consistency?